### PR TITLE
Midpoint cleanup/making cart page more responsive

### PR DESCRIPTION
**Pull request by EthanAuyeung**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Completed midpoint check list for styling edits that needed to be fixed.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
Had trouble with spacing on the cart overview page between  the plus minus button and trash icon and the photo/name of the item this page: 
Screenshot 2024-04-17 at 5 42 34 PM
@BuyankhuuTsCAl can you look at it. Everything else should be done.

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🛍 CC: @EthanAuyeung

### CODE REVIEW

File: src/app/cart/styles.tsx [hunk 1]
```diff
@@ -65,8 +65,9 @@ export const Backtext = styled.p`
 export const TrashIcon = styled(Trash2)`
   width: 30px;
   height: 30px;
-  color: black;
+  color: navy;
```
Comment (BuyankhuuTsCAl) [inline src/app/cart/styles.tsx:68]: wait use the color library

File: src/app/orderConfirmationPickUp/page.tsx [hunk 2]
```diff
@@ -116,10 +115,30 @@ export default function OrderConfirmationPickUp() {
...
+                          Category: {cartItem.category}
+                        </Body2Light>
                       </LabelBox1>
                       <LabelBox>
-                        <Body1Bold>Quantity: {cartItem.quantity}</Body1Bold>
+                        <div>
+                          <span
+                            style={{
+                              display: 'inline-block',
+                              fontWeight: 'bold',
```
Comment (BuyankhuuTsCAl) [inline src/app/orderConfirmationPickUp/page.tsx:127]: try to make these into styled componenet
